### PR TITLE
Housekeeping/Create and fail destination plates

### PR DESCRIPTION
**Pull request by emrojo**
Change event type for plate failed

### CODE REVIEW

File: tests/endpoints/events/biosero/test_destination_failed.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def test_post_destination_completed_missing_barcode(app, client, biosero_auth_he
             "/events",
             data={
                 "user_id": "user1",
-                "event_type": "lh_biosero_cp_destination_failed",
+                "event_type": "lh_biosero_cp_destination_plate_failed",
```
Comment (harrietc52) [inline tests/endpoints/events/biosero/test_destination_failed.py:28]: Could use the constant `EVENT_DESTINATION_FAILED` throughout in these tests

**Review by harrietc52 — APPROVED** [review]

File: tests/endpoints/events/biosero/test_destination_failed.py (hunk 1 above)
Comment (andrewsparkes) [reply tests/endpoints/events/biosero/test_destination_failed.py]: changed all

File: tests/endpoints/events/biosero/test_destination_failed.py [hunk 2]
```diff
@@ -94,7 +95,7 @@ def test_post_event_partially_completed(
                         '{"event": {"uuid": "'
                         + int_to_uuid(1)
                         + (
-                            '", "event_type": "lh_biosero_cp_destination_failed", '
+                            '", "event_type": "' + Biosero.EVENT_DESTINATION_FAILED + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_destination_failed.py:98]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_destination_partial.py [hunk 3]
```diff
@@ -119,7 +120,7 @@ def test_post_event_partially_completed(
                         '{"event": {"uuid": "'
                         + int_to_uuid(1)
                         + (
-                            '", "event_type": "lh_biosero_cp_destination_plate_partial", '
+                            '", "event_type": "' + Biosero.EVENT_DESTINATION_PARTIAL + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_destination_partial.py:123]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_destination_partial_completed.py [hunk 4]
```diff
@@ -175,7 +175,7 @@ def test_post_event_partially_completed(
                             '{"event": {"uuid": "'
                             + int_to_uuid(1)
                             + (
-                                '", "event_type": "lh_biosero_cp_destination_plate_partial_completed", '
+                                '", "event_type": "' + Biosero.EVENT_DESTINATION_PARTIAL_COMPLETED + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_destination_partial_completed.py:178]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_no_pickable_samples.py [hunk 5]
```diff
@@ -72,11 +73,11 @@ def test_post_event_source_no_pickable_samples(
 
                         mocked_rabbit_channel.basic_publish.assert_called_with(
                             exchange="lighthouse.test.examples",
-                            routing_key="test.event.lh_biosero_cp_source_no_pickable_samples",
+                            routing_key=f"test.event.{ Biosero.EVENT_SOURCE_NO_PICKABLE_SAMPLES }",
                             body='{"event": {"uuid": "'
                             + int_to_uuid(1)
                             + (
-                                '", "event_type": "lh_biosero_cp_source_no_pickable_samples", '
+                                '", "event_type": "' + Biosero.EVENT_SOURCE_NO_PICKABLE_SAMPLES + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_no_pickable_samples.py:80]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_no_plate_map_data.py [hunk 6]
```diff
@@ -66,11 +67,11 @@ def test_post_event_no_plate_map_data(
 
                     mocked_rabbit_channel.basic_publish.assert_called_with(
                         exchange="lighthouse.test.examples",
-                        routing_key="test.event.lh_biosero_cp_source_no_plate_map_data",
+                        routing_key=f"test.event.{ Biosero.EVENT_SOURCE_NO_PLATE_MAP_DATA }",
                         body='{"event": {"uuid": "'
                         + int_to_uuid(1)
                         + (
-                            '", "event_type": "lh_biosero_cp_source_no_plate_map_data", '
+                            '", "event_type": "' + Biosero.EVENT_SOURCE_NO_PLATE_MAP_DATA + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_no_plate_map_data.py:74]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_source_completed.py [hunk 7]
```diff
@@ -78,11 +79,11 @@ def test_post_event_partially_completed(
 
                         mocked_rabbit_channel.basic_publish.assert_called_with(
                             exchange="lighthouse.test.examples",
-                            routing_key="test.event.lh_biosero_cp_source_completed",
+                            routing_key=f"test.event.{ Biosero.EVENT_SOURCE_COMPLETED }",
                             body='{"event": {"uuid": "'
                             + int_to_uuid(1)
                             + (
-                                '", "event_type": "lh_biosero_cp_source_completed", '
+                                '", "event_type": "' + Biosero.EVENT_SOURCE_COMPLETED + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_source_completed.py:86]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_source_partial.py [hunk 8]
```diff
@@ -75,11 +76,11 @@ def test_post_event_partially_completed(
 
         mocked_rabbit_channel.basic_publish.assert_called_with(
             exchange="lighthouse.test.examples",
-            routing_key="test.event.lh_biosero_cp_source_partial",
+            routing_key=f"test.event.{ Biosero.EVENT_SOURCE_PARTIAL }",
             body='{"event": {"uuid": "'
             + int_to_uuid(1)
             + (
-                '", "event_type": "lh_biosero_cp_source_partial", '
+                '", "event_type": "' + Biosero.EVENT_SOURCE_PARTIAL + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_source_partial.py:83]: An f-string here might be a little clearer?

File: tests/endpoints/events/biosero/test_source_unrecognised.py [hunk 9]
```diff
@@ -48,11 +49,11 @@ def test_post_event_source_unrecognised(
 
                     mocked_rabbit_channel.basic_publish.assert_called_with(
                         exchange="lighthouse.test.examples",
-                        routing_key="test.event.lh_biosero_cp_source_plate_unrecognised",
+                        routing_key=f"test.event.{ Biosero.EVENT_SOURCE_UNRECOGNISED }",
                         body='{"event": {"uuid": "'
                         + int_to_uuid(1)
                         + (
-                            '", "event_type": "lh_biosero_cp_source_plate_unrecognised", '
+                            '", "event_type": "' + Biosero.EVENT_SOURCE_UNRECOGNISED + '", '
```
Comment (pjvv) [inline tests/endpoints/events/biosero/test_source_unrecognised.py:56]: An f-string here might be a little clearer?

**Review by pjvv — APPROVED** [review]

**Review by emrojo — COMMENTED** [review]
All good

**Pull request merged**